scripts/newinstall.sh

```diff
@@ -187,11 +187,12 @@ fi
 
 if true; then
 	# Check the version by running a small Python program (taken from the Python EUPS package)
+    # XXX should not assume that python is in the default path
```

Comment:
Yes we should. Please remove this comment. We've talked about this extensively in the past. $PYTHON is not correct here because we are checking that the python in their path is okay for building the stack (not good enough for running eups).